### DISCUSSION TITLE
test-basic.py test case 1 // 0

### DISCUSSION THREAD

**Discussion by cohogainLYIT** [pallets/flask#4493] [Q&A] [answered]
Can I ask what the purpose of the division of 1 and 0 is for below and can be found in other test cases in test_basic.py? Is it just a condition you never hope to encounter? Is this the appropriate syntax for such a case?

def test_exception_propagation(app, client):
    def apprunner(config_key):
    @app.route("/")
         def index():
             1 // 0

**Comment by davidism** [comment, marked as answer]
Look at what the test is testing: it expects that a view that raises an unhandled exception will produce a specific result. ``1/0`` is one easy way to cause an exception.

**Reply by cohogainLYIT** [reply]
I see that now, thank you
